main/Helper.cpp

```diff
@@ -526,50 +526,54 @@ std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, i
 	return ret;
 }
 
-//convert date string 10/12/2014 10:45:58 en  struct tm
-void DateAsciiTotmTime (std::string &sTime , struct tm &tmTime  )
+std::string TimeToString(time_t ltime, _eTimeFormat format)
```

Comment:
No, we're not supplying a time struct, just time_t (int), but I guess the naming should be different to avoid confusing... I started this with a struct, which is why it's still named ltime. Anyway, there is no need for a pointer in this case.

You can now just eg. use the time() (or mytime() for that matter) function to get the time as an integer and do your calculations on that and directly get a time as string. Otherwise you need to construct and supply a the struct every single time (it's part of the idea for this function, do the constructing and converting to string transparently).